EFDataAccessLibrary/Interfaces/IContactRepository.cs

```diff
@@ -8,14 +8,16 @@ namespace EFDataAccessLibrary.Interfaces
 	public interface IContactRepository : IGenericRepository<Contact>
 	{
 		List<Contact> GetAllContacts();
-		void SaveChanges();
-		Task SaveChangesAsync();
+		Task<List<Contact>> GetAllContactsAsync(CancellationToken cancellationToken);
+        void SaveChanges();
```

Comment:
immer en visibility modifyer definiere. also private, public, protected

in dem fall warschinlich public

Reply:
> immer en visibility modifyer definiere. also private, public, protected
> 
> in dem fall warschinlich public

Ih allne bispil vo interfaces sinds ebe ohni visibility modifiers gsi. Bisch sicher dass au ih demfall das nötig wär? Macht nöd so sinn dass ih de offizielle documentation vo microsoft das ohni drin wär & au susch ih dene spezifische interface files es nie mit vis. mods. gseh